### ISSUE TITLE
Long DESCRIPTION has embedded newline

### ISSUE THREAD

**Issue by Azhrei** [ics-py/ics-py#210]
I'm using ics v0.4.  Output of executing `python3` interactively:
```
Python 3.7.1 (default, Dec 14 2018, 13:28:58) 
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
```
The issue I'm having is that a long **DESCRIPTION** that spans multiple lines in the `.ics` file is not having the embedded newlines removed when it should (example event is below).

(I will grab the latest v0.6 but I don't see my issue mentioned in an issue or PR since v0.4, so I'm thinking it hasn't been addressed.)

Basically, the iCal format allows for embedded newlines that _should be stripped from the data_ if the following line is indented by one or more spaces.  Both the newline and the spaces are to be removed.  (Basically, this works just like headers in an email message or HTTP request.  What a surprise, huh? 🙂)

I might have a workaround for my code by replacing `\n ` (newline-space) with an empty string, but I'm not sure that'll work in all cases.

Attachment renamed to `buggy.txt` because of GitHub...
[buggy.ics](https://github.com/C4ptainCrunch/ics.py/files/3883235/buggy.txt)

**Comment by N-Coder** [comment]
This is probably the same issue as in #215. According to the discussion there, unfolding while parsing should already be supported.

**Comment by N-Coder** [comment]
This has been fixed and e5ed70e91fc3a431ae95eef746bcbb68f48294f1 added a test case preventing regressions in [tests/issues/gh210.py](https://github.com/ics-py/ics-py/commit/e5ed70e91fc3a431ae95eef746bcbb68f48294f1#diff-9efaf0a725fa860d6e74c9124ac7bb7a8e16aadc278df3c6b426c195c9ba3f4e).
